sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/ChangeFeedProcessorOptions.java

```diff
@@ -376,7 +376,14 @@ public ChangeFeedProcessorOptions setScheduler(Scheduler scheduler) {
     }
 
     /***
-     * Set the feed poll throughput control config.
+     * Set the feed poll local throughput control config.
+     *
+     * Please use this config with cautious. By default, CFP will try to process the changes as fast as possible,
+     * only use this config if you want to limit the RU can be used for your change feed processing.
+     * By using this config, it can possibly slow down the process and causing the lag.
+     * 
+     * For direct mode, please configure the throughput control group with the (total RU you would allow for changeFeed processing).
+     * For gateway mode, please configure the throughput control group with the (total RU you would allow for changeFeed processing / total CFP Instances).
```

Comment:
```suggestion
     * For gateway mode, please configure the throughput control group with the total RU you would allow for changeFeed processing / total CFP Instances.
```